repository/Seaside-Core.package/WARemoteAddressProtectionFilter.class/instance/remoteAddressFromRequest..st

```diff
@@ -1,11 +1,6 @@
 private
 remoteAddressFromRequest: aRequest
-	remoteAddress ifNil: [
-		remoteAddress := aRequest
-			headerAt: 'x-forwarded-for'
-			ifAbsent: [
-				aRequest
-					headerAt: 'forwarded-for'
-					ifAbsent: [
-						aRequest remoteAddress ] ] ].
-	^ aRequest remoteAddress
\ No newline at end of file
+	| result |
+	result := aRequest forwarded ifNil:[ aRequest remoteAddress ].
+	remoteAddress ifNil:[ remoteAddress := result ].
```

Comment:
Why do we need to check `#forwarded` every time? Can it change?

Reply:
Not sure I understand what you mean. We need to check the remoteAddress (or the value in the forwarded header) in every request, no?

Reply:
Are you thinking of mobile clients?

Reply:
not really, the filter just needs to verify the value in every request. It sets the value it extracts from the first request it processes and requires any subsequent request to have the same value.

I do not intend to improve this filter functionally, just fix the code.